thetis/timezone.py

```diff
@@ -0,0 +1,43 @@
+"""
+Timezone definitions and conversion methods
+"""
+import datetime
+import pytz
+
+utc_tz = pytz.timezone('UTC')
```

Comment:
There's also pytz.utc - do we really need our own?

Reply:
Nope. Using `pytz.utc` now.